api/django-fileupload/django_fileupload/views.py

```diff
@@ -59,7 +60,10 @@ def create(self, request, *args, **kwargs):
                     raise ValidationError(_("Files with incorrect extension in the request."))
                 response = []
                 with exclusive_insert_table_lock(FileUploadBatch):
-                    file_upload_batch = FileUploadBatch.objects.create(owner=request.user)
+                    if self.has_owner:
+                        file_upload_batch = FileUploadBatch.objects.create(owner=request.user)
+                    else:
+                        file_upload_batch = FileUploadBatch.objects.create()
```

Comment:
I think it is a good idea to always have an upload owner. Even just knowing that a batch of files was uploaded by an anonymous user is better than no information at all.

Reply:
Could you provide a suggestion for how to handle this? Wouldn't a file owner of "none" signal that the owner is anonymous?

Reply:
I ended up reintroducing this check as the owner has to be a user or empty.